tests/EndToEnd/member-update/MemberSubscribeCest.php

```diff
@@ -60,19 +60,29 @@ public function testMemberNameAndEmailUpdatedWhenChangedInMemberMouse(EndToEndTe
 		$newFirstName    = 'New First Name';
 		$newLastName     = 'New Last Name';
 		$newEmailAddress = 'new-' . $emailAddress;
-		$I->memberMouseUpdateMember($I, $emailAddress, $newEmailAddress, $newFirstName, $newLastName);
+		$I->memberMouseUpdateMember(
+			I: $I,
+			emailAddress: $emailAddress,
+			newEmailAddress: $newEmailAddress,
+			newFirstName: $newFirstName,
+			newLastName: $newLastName
+		);
 
 		// Check the subscriber's email address was updated in ConvertKit.
-		$subscriberAfterNewEmailAddress = $I->apiCheckSubscriberExists($I, $newEmailAddress, $newFirstName);
+		$subscriberAfterNewEmailAddress = $I->apiCheckSubscriberExists(
+			I: $I,
+			emailAddress: $newEmailAddress,
+			firstName: $newFirstName
+		);
 
 		// Confirm the subscriber ID is the same.
 		$I->assertEquals($subscriber['id'], $subscriberAfterNewEmailAddress['id']);
 
 		// Check that the subscriber has the custom field data.
 		$I->apiCustomFieldDataIsValid(
-			$I,
-			$subscriberAfterNewEmailAddress,
-			[
+			I: $I,
```

Comment:
"I" as a named fields doesn't add much information - my vote would be to just pass them as normal params.